src/openforms/forms/tests/e2e_tests/test_registration_backend_conf.py

```diff
@@ -142,40 +144,57 @@ def collect_requests(request):
             str(zgw_api_2.pk),
         )
 
-    async def test_configuration_objects_api_group(self):
+    @patch(
+        "openforms.registrations.contrib.objects_api.client.ObjecttypesClient.list_objecttype_versions"
+    )
+    @patch(
+        "openforms.registrations.contrib.objects_api.client.ObjecttypesClient.list_objecttypes"
+    )
```

Comment:
yeah, the API interaction is tested separately and combining VCR with `async` in tests + playwright & needing to have a docker-compose set up for all this is just... too much. It's one thing if this API interaction is what is being tested, but another thing if it's just some data that needs to be provided to prevent our "server" returning HTTP 500 errors in playwright (which aren't fatal).

Essentially it's a quick cleanup so that the next time someone needs to debug this test, they're not sent in the wrong direction because HTTP 500s are being returned (as happened to me :( )